### PR TITLE
added esConsumes to modules in EventFilter/SiStripRawToDigi

### PR DESCRIPTION
**Pull request by JamminJones**
#### PR description:

added esConsumes to 2 modules
#### PR validation:

the code compiles

### CODE REVIEW

**Comment by JamminJones** [comment]
SiStripRawToDigiModule.cc was skipped due to complexity.

**Comment by Dr15Jones** [comment]
This change was made under my supervision.

**Comment by Dr15Jones** [comment]
please test

File: EventFilter/SiStripRawToDigi/plugins/SiStripDigiToRawModule.h [hunk 1]
```diff
@@ -51,6 +54,7 @@ namespace sistrip {
     edm::EDGetTokenT<edm::DetSetVector<SiStripDigi> > tokenDigi;
     edm::InputTag rawDataTag_;
     edm::EDGetTokenT<FEDRawDataCollection> tokenRawBuffer;
+    edm::ESGetToken<SiStripFedCabling, SiStripFedCablingRcd> tokenCabeling;
```
Comment (slava77) [inline EventFilter/SiStripRawToDigi/plugins/SiStripDigiToRawModule.h:57]: ```tokenCabling``` to match the underlying type names

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/30475	af127a6
- code changes are in line with the PR description and the follow up review
- jenkins tests pass and comparisons with the baseline show no differences, as expected

**Comment by silviodonato** [comment]
+1

**Pull request merged**